app/package.json

```diff
@@ -26,106 +26,108 @@
     "flask",
     "blockchain"
   ],
-  "author": "Nick Williams",
+  "author": "Sempo Pty Ltd",
   "license": "GPL-3.0",
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/preset-env": "^7.9.5",
-    "@babel/preset-react": "^7.9.4",
-    "@types/chart.js": "^2.8.9",
-    "@types/jest": "^25.2.1",
-    "@types/node": "^13.1.4",
-    "@types/react": "^16.8.6",
+    "@babel/cli": "^7.10.5",
+    "@babel/core": "^7.11.0",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/preset-stage-2": "7.8.3",
+    "@types/chart.js": "^2.9.23",
+    "@types/jest": "^26.0.8",
+    "@types/node": "^14.0.27",
+    "@types/react": "^16.9.44",
     "@types/react-qr-reader": "^2.1.3",
-    "@types/react-redux": "^7.1.8",
+    "@types/react-redux": "^7.1.9",
     "@types/react-responsive": "^8.0.2",
     "@types/react-router-dom": "^5.1.5",
-    "@types/redux-form": "^8.2.5",
+    "@types/redux-form": "^8.2.7",
     "@webpack-cli/migrate": "^0.1.9",
     "awesome-typescript-loader": "^5.2.1",
-    "babel-core": "^6.26.3",
-    "babel-jest": "^25.1.0",
-    "babel-loader": "^7.1.2",
+    "babel-cli": "^6.26.0",
+    "babel-jest": "^26.2.2",
+    "babel-loader": "^8.1.0",
     "babel-plugin-import": "^1.13.0",
     "babel-plugin-transform-export-extensions": "^6.22.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
+    "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-2": "^6.24.1",
-    "clean-webpack-plugin": "^0.1.19",
-    "compression-webpack-plugin": "^2.0.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "compression-webpack-plugin": "^4.0.0",
     "css-loader": "^3.6.0",
```

Comment:
Can't upgrade to v4+ due to https://github.com/seek-oss/css-modules-typescript-loader/issues/39#issue-667187503